sale_stock_mto_as_mts_orderpoint_bom/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    "name": " - BOM",
+    "summary": (
+        "Integration module between sale_stock_mto_as_mts_orderpoint and mrp"
+    ),
+    "version": "14.0.1.0.0",
+    "category": "Operations/Inventory/Delivery",
+    "website": "https://github.com/OCA/stock-logistics-workflow",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "maintainers": ["mmequignon"],
+    "license": "AGPL-3",
+    "installable": True,
+    "auto_install": True,
+    "depends": [
+        "sale_stock_mto_as_mts_orderpoint",
+        "mrp",
+        "mrp_helper",
```

Comment:
Module was renamed https://github.com/OCA/manufacture/pull/1350
```suggestion
        "mrp_bom_find_ignore",
```
You need to add it in the test requirement